spec/features/user_can_edit_posts_spec.rb

```diff
@@ -37,20 +37,8 @@
     create_user_two
     login_user_two
     visit '/posts'
-    expect(page).to have_no_button("Edit")
-  end
-
-  scenario "User cannot delete another user's post" do
```

Comment:
Is this meant to be deleted?

Reply:
This was moved to user_can_delete_posts_spec